src/instance/minecraft/commands/CataCommand.ts

```diff
@@ -6,7 +6,7 @@ import { Client, SkyblockMember } from 'hypixel-api-reborn'
 const Mojang = require('mojang')
 
 export default {
-  triggers: ['catacomb', 'cata'],
+  triggers: ['catacomb', 'cata', 'c'],
```

Comment:
This will conflict with Calculate command.

Reply:
Whoops, didn't think about that